### ISSUE TITLE
Review "read more" style links to include full context

### ISSUE THREAD

**Issue by madcampos** [techtankto/techtank-site#151]
[SC 2.4.4](https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-in-context.html) states that links should be readable regardless of the context around them.

This rule exists because screen readers have the ability to navigate a page by links only. This makes so a user can jump between multiple links in a page without necessarily having the context of the content around the link.

To solve this, links should do one of those things:
- Visibly include the content it refers to.
- Include the content it refers to as an `.sr-only` text.

E.g.:
```html
<card-component>
  <header>
    <h2>Title of a blog post</h2>
  </header>
  <!-- ... -->
  <footer>
    <a href="/link/to/blog/post">
      <span>Read more</span>
      <sr-only>about "Title of a blog post"</sr-only>
      <span>...</span>
    </a>
  </footer>
</card-component>
```

This will display the text "Read more..." visually, but contain the context for screen readers.
